### DISCUSSION TITLE
Is WKWebView still needed on IOS?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#15176] [CLI - Capacitor / Cordova modes]
Here: https://github.com/apache/cordova-plugin-wkwebview-engine 

I read 

> This plugin is now deprecated as it's implementation has been moved to cordova-ios platform making this plugin redundent. When migrating to cordova-ios@6, you should remove this plugin.

So I kindly ask to update the doc here https://quasar.dev/quasar-cli-vite/developing-cordova-apps/preparation#switching-to-ios-wkwebview to reflect if is still neded or otherwise please remove this section

**Comment by metalsadman** [comment]
please P.R. if you can find time. ty.

**Reply by realtebo** [reply]
Sorry, I'm here to ask if which of the 2 infos are right, so I cannot make a PR because only quasar dev team can give the correct answer

**Reply by metalsadman** [reply]
that info is obviously correct as it come from apache when using latest cordova version. i would suggest capacitor anyhow as most cordova plugins will work on it anyway, provides better api and don't have to worry bout using whatever webview.
